tests/integration/test_config.py

```diff
@@ -282,3 +283,39 @@ def test_parse_special_tasks_families(flow, scheduler, validate, section):
             'foo(P1D)',
             'foot(P1D)'
         }
+
+
+def test_queue_treated_as_implicit(flow, validate):
+    """Tasks listed in queue should be regarded as implicit
```

Comment:
I'm not sure this docstring makes sense. It's more that any implicit task - i.e. a task name with no definition under runtime - assigned to a queue will cause validation failure (or, better, just a warning as suggested above).